fendermint/vm/topdown/src/sync/mod.rs

```diff
@@ -21,6 +21,14 @@ use fendermint_vm_genesis::{Power, Validator};
 
 pub use syncer::fetch_topdown_events;
 
+#[derive(Clone)]
+pub enum TopDownSyncEvent {
```

Comment:
Can you elaborate more on this? it's unclear how it may be produced.

Reply:
The following PRs are explained more extensively. I will add some doc to this.